src/leap/bitmask/gui/logwindow.py

```diff
@@ -72,36 +132,45 @@ def __init__(self, parent, handler):
         self._current_filter = ""
         self._current_history = ""
 
-        # Load logging history and connect logger with the widget
-        self._logging_handler = handler
-        self._connect_to_handler()
+        self._set_logs_to_display()
+
+        self._my_handler = QtLogHandler(format_string=LOG_FORMAT)
+
         self._load_history()
+        self._connect_to_handler()
```

Comment:
Not exactly, Qt docs talks about handlers but they are the methods that handles the connected signals.
In this context, the handler refers to the `logbook.Handler` that happens to have some Qt stuff on it.

That method is the result of some refactoring here and there and right now is just one line so I'll move it below the `my_handler` creation to simplify things :)